cachi2/core/package_managers/generic.py

```diff
@@ -0,0 +1,33 @@
+from cachi2.core.errors import PackageRejected
```

Comment:
@brunoapimentel Do we want to split it to multiple modules under `generic` dir or just one module is fine ?

Reply:
This is kept as experimental for now. If the code grows more than we expected, I'd say we should break it down.